playbooks/robusta_playbooks/pod_enrichments.py

```diff
@@ -13,12 +15,39 @@ def pod_graph_enricher(pod_event: PodEvent, params: ResourceGraphEnricherParams)
         "pod": pod_event.get_pod().metadata.name,
         "namespace": pod_event.get_pod().metadata.namespace,
     }
+    pod = pod_event.get_pod()
```

Comment:
I would add it at the beginning of the function, and use it in the lines above as well